.github/workflows/dependent-issues.yml

```
@@ -0,0 +1,36 @@
+---
+name: Dependency Checks
```

```suggestion
name: PR Dependency Checks
```

Maybe? To be clear that this isn't meant to group the many other types of dependency-related checks we do?

---

this along with L22 on GH UI will be 

```Dependency Checks/PR Dependency```

COmbined it makes more sense

---

We should likely also rename the file to match the workflow name.

---

> this along with L22 on GH UI will be
> 
> `Dependency Checks/PR Dependency`
> 
> COmbined it makes more sense

But in some places, like the main GH Actions UI, it will only show the workflow name until you drill down. This workflow should be much more narrowly scoped than about "dependency checks" in general, it should be about PR dependencies.